Examples/Physics_applications/free_electron_laser/plot_sim.py

```diff
@@ -0,0 +1,52 @@
+import matplotlib.pyplot as plt
+from openpmd_viewer import OpenPMDTimeSeries
+
+ts = OpenPMDTimeSeries("./diags/diag_labframe/")
```

Comment:
Just wondering if we should pass this file path as a command line argument as well and read it through `sys.argv[1]`, like in the analysis script, in case someone runs this manually outside of CTest and moves files around while doing so...

Reply:
I am OK leaving it as is for now, but then changing the `plot.py` in this example as well as other examples in a follow PR, if we think this is how users should run it.